.emacs.d/test/lisp/run-tests.el

```diff
@@ -4,4 +4,17 @@
 
 (load-file (concat (file-name-directory load-file-name) "/dot-tests.el"))
 
+(when (require 'undercover nil 'noerror)
+  ;; undercover can't search for matching files recursively
+  ;; override a local function that fixes this
+  (advice-add
+   #'undercover--wildcards-to-files :override
+   (lambda (wildcards)
```

Comment:
Gotcha, does the argument need to be the same though?  (Like, can you do `(lambda (_)` on this line instead?  Or `(_wildcards)`?)  I haven't tested this but we might get lexical binding errors, see [EL manual § 11.9.4 "Using Lexical Binding"](https://www.gnu.org/software/emacs/manual/html_node/elisp/Using-Lexical-Binding.html):
> (To silence byte-compiler warnings about unused variables, just use a variable name that start with an underscore. The byte-compiler interprets this as an indication that this is a variable known not to be used.)

If it's not used, changing it to underscore would be nice.